os-eap-sso/added/keycloak.sh

```diff
@@ -319,6 +320,12 @@ function configure_subsystem() {
             deployments=`echo "${deployments}" | sed "s|##SSO_SAML_LOGOUT_PAGE##|/|" `
           fi
 
+          if [ -n "$SSO_PRINCIPAL_ATTRIBUTE" ]; then
```

Comment:
Can you just turn this into a one-liner by using `${SSO_PRINCIPAL_ATTRIBUTE:-preferred_username}`?

Reply:
Done. I was just trying to follow the same style as for the rest.
Thanks for the review

Reply:
ahh, understood.